weatherman.py

```diff
@@ -0,0 +1,109 @@
+"""
+Weatherman
+This module generates yearly reports on weather data collected from  Dec 1996 - May 2011.
+Report no. 1 gives the yearly max/min temperature and max/min humidity.
+Report no. 2 gives the hottest day of the year and the temperature on that day.
+"""
+
+import os
+import sys
+
+import argparse
+
+
+max_temp_ = {}
+max_temp_date = {}
+min_temp_ = {}
+max_humidity= {}
+min_humidity= {}
+
+parser = argparse.ArgumentParser(prog = 'weatherman',
```

Comment:
Parser code should be in driver function.

Reply:
Right. Moving.
Done.